### PR TITLE
DROOLS-5299 DMN strongly typed javadoc/file comment disclaimer

### PR DESCRIPTION
**Pull request by tarilabs**

### CODE REVIEW

File: kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/typesafe/DMNInputSetType.java [hunk 1]
```diff
@@ -93,4 +95,20 @@ public void initFields() {
     public List<FieldDefinition> findInheritedDeclaredFields() {
         return Collections.emptyList();
     }
+
+    public void initJavadoc(DMNModelImpl dmnModel, String disclaimerMarker) {
+        StringBuilder sb = new StringBuilder("A representation of all the InputData and other DRG Requirement of the whole DMN '").append(dmnModel.getName()).append("' inputs.").append("<br/>\n");
+        sb.append("<br/>\n");
+        sb.append("This has been automatically generated from the following DMN asset.").append("<br/>\n");
```
Comment (lucamolteni) [inline kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/typesafe/DMNInputSetType.java]: as the message is similar, can't we put it in a single method? I wouldn't want to change it in three places

**Review by lucamolteni — APPROVED** [review]

File: kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/typesafe/DMNInputSetType.java (hunk 1 above)
Comment (tarilabs) [reply kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/typesafe/DMNInputSetType.java]: the part which is actually indeed the same, is already factored out.
But the messages are different.

Comment (tarilabs) [reply kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/typesafe/DMNInputSetType.java]: nevermind, there is no way to keep this message and please SonarCloud, refactoring..

Comment (lucamolteni) [reply kie-dmn/kie-dmn-core/src/main/java/org/kie/dmn/typesafe/DMNInputSetType.java]: lol

**Comment by tarilabs** [comment]
the "duplications" are 1 Setter and 1 Getter :/

**Comment by tarilabs** [comment]
```
18:56:44 [INFO] --- spotbugs-maven-plugin:3.1.8:spotbugs (spotbugs) @ business-monitoring-webapp ---
18:56:44 [INFO] Fork Value is true
18:56:46 Build timed out (after 91 minutes). Marking the build as aborted.
18:56:46 Build step 'Build dependent downstream repositories (for PR builds)' marked build as failure
18:56:46 Terminating xvnc.
18:56:46 $ vncserver -kill :70
18:56:46 Killing Xvnc process ID 9914
18:56:47 [INFO] 
18:56:47 [INFO] ------------------------------------------------------------------------
18:56:47 [INFO] Skipping Business Central - Monitoring Distribution Wars
18:56:47 [INFO] This project has been banned from the build due to previous failures.
18:56:47 [INFO] ------------------------------------------------------------------------
18:56:47 Skipping publisher since build result is FAILURE****
```

compile downstream unrelated

**Review by hellowdan — APPROVED** [review]

**Pull request merged**